applications/TradeCoin.md

```diff
@@ -0,0 +1,198 @@
+
+### Ecosystem Fit
+
+We take inspiration from the latest trend in the blockchain ecosystem which we aim to replicate on polkadot. Telegram bots have gained popularity in the crypto market, with a total market capitalization of [$303 million](https://www.dlnews.com/articles/defi/unibot-telegram-bot-unveils-cloud-service-for-private-keys/).
+
+For example, Unibot is a Telegram bot that allows users to trade cryptocurrencies without leaving the Telegram app. Unibot offers various features, such as decentralized copy trading, limit orders on decentralized exchanges (DEXs), protection against MEV (Miner Extractable Value) bots, and more. The platform has gained momentum and has shown significant growth in trade volume, revenue, and user counts over the last two months. Unibot also has its native token called UNIBOT, with holders receiving a portion of transaction fees and trading volume.
+
+The UNIBOT has seen substantial user growth and has facilitated trades worth $54 million, distributing nearly $1 million in revenue back. Holders of UNIBOT tokens receive rewards of 40% of transaction fees and 1% of the platform's total trading volume.
+
+The appeal of Unibot and other Telegram-based trading bots lies in their user-friendly interface, providing ease of use compared to decentralized exchanges like Uniswap. With Unibot, traders can execute market buy/sell orders through a simple Telegram command, eliminating the need for continuous wallet logins and cross-checking token information. This streamlined process is particularly advantageous for fast swaps and scalping strategies, attracting traders who value efficiency and.
+
+Unibot's success has received endorsement from the crypto community, with popular crypto traders on Twitter praising the platform's user experience and its advantage of trading directly through Telegram for various trading strategies. As a result, newer tokens like WagieBot, 0xSniper, Bridge, and Bolt have experienced significant growth, emulating Unibot's trajectory.
+
+Furthermore, in TradeCoin we improve on private keys storage location by providing an opensource solution.
+[CertiK](https://www.dlnews.com/articles/defi/unibot-telegram-bot-unveils-cloud-service-for-private-keys/), a blockchain security firm, raises concerns about the lack of information provided by Unibot regarding the storage location and encryption method of private keys. The involvement of a third-party provider could potentially pose a risk such as susceptibility to supply chain attacks.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Alexandr Fedotov
+
+### Contact
+
+- **Contact Name:** Alexandr Fedotov
+- **Contact Email:** alexkekfedotov@gmail.com
+- **Website:** [trade-coin-xi.vercel.app](https://trade-coin-xi.vercel.app)
+
+### Legal Structure
+
+Individual
```

Comment:
Please provide a name and registered address. If you prefer not to do this publicly, please [email us](mailto:grants@web3.foundation) with reference to this pull request.